src/plugins/vis_default_editor/public/_default.scss

```diff
@@ -1,45 +1,25 @@
 .visEditor--default {
+  height: 1px;
```

Comment:
It would be helpful to have a comment here explaining the reason as it will undoubtedly stand out to future reviewers of this file.